### PR TITLE
Test the show-deployments CLI command, and fix a bug

### PR DESCRIPTION
**Pull request by alexwlchan**
Turns out `show-deployments` got busted in a recent refactoring. Now it's fixed, and tested! (Sort of – it checks the lines are broadly correct, but I don't check the exact output because that feels brittle.)

### CODE REVIEW

File: src/deploy/project.py [hunk 1]
```diff
@@ -282,19 +282,18 @@ def _create_release(self, description, images):
             "deployments": []
         }
 
-    def get_deployments(self, release_id, limit, environment_id):
```
Comment (kenoir) [inline src/deploy/project.py]: Why are you removing the environment filter?

Comment (alexwlchan) [reply src/deploy/project.py]: It wasn’t exposed anywhere I could find in the CLI? Unless I missed something?

I can put it back if it was meant to be in the CLI.

Comment (kenoir) [reply src/deploy/project.py]: It looks like the filter was removed here: https://github.com/wellcomecollection/weco-deploy/commit/e0d4dffb9a47a88942d12798ee0fe4beb910c35c#diff-48252ce381e5fcfcff5ddf9bc65a66cd33f6c84e8d4028525727ec1f212d3741L469 after being added here: https://github.com/wellcomecollection/weco-deploy/commit/793de413f8a2320528f377214d51d988a230a4ab#diff-48252ce381e5fcfcff5ddf9bc65a66cd33f6c84e8d4028525727ec1f212d3741

Comment (alexwlchan) [reply src/deploy/project.py]: Oops. :see_no_evil:

I'll put it back!

Comment (kenoir) [reply src/deploy/project.py]: I think perhaps it was removed in the CLI accidentally? I don't like the way that filter works, but it is valuable to be able to filter where there are large volumes of stage deploys swamping out prod.

Comment (kenoir) [reply src/deploy/project.py]: As far as merging this PR goes I think it's a separate issue though.

**Review by kenoir — APPROVED** [review]

**Pull request merged**